components/css/buckyless/header.less

```diff
@@ -67,6 +54,20 @@ portal-header {
       .menu-toggle.md-button {
         min-width: 48px;
         margin: 0;
+
+        + .title-link h1 {
+          @media (min-width: @xs) {
+            margin-left: 0;
+          }
+        }
+
+        &.hide-gt-xs {
+          + .title-link h1 {
```

Comment:
:question: would a [general sibling selector](https://developer.mozilla.org/en-US/docs/Web/CSS/General_sibling_selectors) be more robust?